### ISSUE TITLE
what are our options for signing out of GooglePlayGames in v11.0.1

### ISSUE THREAD

**Issue by actionthom** [playgameservices/play-games-plugin-for-unity#3152]
With PlayGamesPlatform.Instance.SignOut being removed, does version 11.0.1 contain the google sign in client, to allow us to sign out if we want to? And if not is there a recommended solution for people who still want to allow users to sign out?

**Versions**
- Unity version: 2020.3.33f1
- Google Play Games Plugin for Unity version: 11.0.1

**Additional context**
Our product allows login and saving via both GooglePlayGames and Facebook and we allow the user to choose. We allow the user to switch accounts in-game so they need to be able to log out of one and in to another

**Comment by MansyrevAY** [comment]
Yes, and how should people switch their accounts if they want to?

**Comment by rico345100** [comment]
I'm having a very serious issue that the authentication callback never fired on some random people, so I was going to upgrade the version to the latest, hoping it's solved.

But after I read the upgrading guide, I changed my mind about not updating it because the SignOut method has been removed. I couldn't understand why it was removed, and found a guy asked about the reason:
https://github.com/playgameservices/play-games-plugin-for-unity/issues/3138

So the reason of removed this feature is just that now people can sign out themself, outside of the app?

Am I the only one who seems too much complicated in the process of logging out, to tell the people to sign out in that way?

**Comment by ArjunIsCool** [comment]
this is so fucking annoying, i spent hours trying to figure out whats wrong in my code when i hit upon this page. I cant believe they really removed it. It made logging in and out so much easier.

**Comment by Zarkend** [comment]
The support on this package is astonishing, holy moly

**Comment by MoHortus** [comment]
lol so there is no option to deactive or logout etc. amazing.

wanted to make for my app that user can manually enable/disable Play Games Service while being in the game. I guess i will have to write a note: please restart the game on disable 🤡

**Comment by MoHortus** [comment]
haha now I see the whole issue. Even though I have switch in the game, which would run NO code related to Google Play Games, so not running for example:
PlayGamesPlatform.Activate()
PlayGamesPlatform.Instance.Authenticate

there is still active login to Google Play Games when I start the app, ... well I will have to try v10.14 I guess. Never had I used last version of plugin and reverted to some older version.

So I totally agree with the above: why would somebody call in the documentation that it is improvements for developers to have no option to sign out (or actually to have forced sign in) ...

**Comment by ArjunIsCool** [comment]
I would say that this was done purposely to force users to use Google Play
Games all the time. Very creepy tho.

**Comment by PissedPawn** [comment]
So guys what happens if the user signs in with the wrong google account and wants to switch it back? is there a way to get around that?

**Comment by ArjunIsCool** [comment]
Apart from resetting the game data or reinstalling, not really :/

**Comment by MackowiakMateusz** [comment]
...

**Comment by exzizt** [comment]
We really need a way to sign out, please. I even changed my current profile in the Google Play Games app, but it's still logging me in with the initial one.

**Comment by PissedPawn** [comment]
I just downgraded and used 10.14 instead. Worked like a charm. I don't see a reason to use 11 with all of these shortcomings it has.

**Comment by hippogamesunity** [comment]
"The sign out method is removed, and we will no longer require an in-game button to sign in or sign out of Play Games Services."

**Comment by wxyz24** [comment]
latest version with an easy switch https://developer.android.com/games/pgs/app-shortcuts
